### PR TITLE
ParseableInterface: improve test coverage on Windows

### PR DESCRIPTION
**Pull request by compnerd**
This slightly improves the passing set of tests on Windows.

Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN).

### CODE REVIEW

**Comment by compnerd** [comment]
CC: @jrose-apple

**Comment by compnerd** [comment]
@swift-ci please test

File: test/ParseableInterface/synthesized.swift [hunk 1]
```diff
@@ -11,13 +11,21 @@ public enum HasRawValue: Int {
   // CHECK-DAG: }
 } // CHECK: {{^}$}}
 
-// CHECK-LABEL: @objc public enum ObjCEnum : Int {
+#if os(Windows) && (arch(arm64) || arch(x86_64))
+@objc public enum ObjCEnum: Int32 {
```
Comment (jrose-apple) [inline test/ParseableInterface/synthesized.swift]: Oh, just change this to Int32 everywhere, then. The Int-ness isn't important.

Comment (compnerd) [reply test/ParseableInterface/synthesized.swift]: I like simpler!

**Comment by compnerd** [comment]
@swift-ci please test

File: test/ParseableInterface/synthesized.swift [hunk 2]
```diff
@@ -11,13 +11,15 @@ public enum HasRawValue: Int {
...
+}
+
+// CHECK-LABEL: @objc public enum ObjCEnum : Int32 {
+// CHECK-NEXT: case a, b, c
+// CHECK-DAG: public typealias RawValue = Swift.Int32
+// CHECK-DAG: @inlinable public init?(rawValue: Swift.Int32)
+// CHECK-DAG: public var rawValue: Swift.Int32 {
+// CHECK-DAG:   @inlinable get{{$}}
+// CHECK-DAG: }
+// CHECK: {{^}$}}
```
Comment (jrose-apple) [inline test/ParseableInterface/synthesized.swift]: Oh, can you put this back now that it no longer has to be split up?

**Pull request merged**